src/index.html

```diff
@@ -22,6 +22,316 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="list">
+          <li>
+            <a
+              href="#"
+              class="list__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="list__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="list__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="list__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="list__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="list__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="list__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="list__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__picture"
+          src="./images/imac.jpeg"
+          alt="iMac"
+        />
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">Product code: 195434</div>
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars__star-1"></div>
+            <div class="stars__star-2"></div>
+            <div class="stars__star-3"></div>
+            <div class="stars__star-4"></div>
+            <div class="stars__star-5"></div>
+          </div>
+          Reviews: 5
+        </div>
+        <div class="card__price">Price:</div>
+        <a
+          class="card__button"
+          href="#"
+          data-qa="card_hover"
```

Comment:
The 'data-qa' attribute is used here. Ensure that this is necessary for your testing or development process, as it does not affect the visual output.